components/feature/qr/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <style name="Theme.Mozac.Qr" parent="Theme.AppCompat.Light.DarkActionBar">
+        <item name="android:background">@android:color/background_dark</item>
+    </style>
+</resources>
```

Comment:
This part is a bit confusing to me: We are creating this theme to set the background on the `RelativeLayout`. There's technically no difference between setting the `background` on the `RelativeLayout` directly.

Is this what caused the `Overdraw` lint warning? In this case we probably can just ignore it inside `fragment_layout.xml` with something like `tools:ignore=".."`.

Reply:
Yeah, was just trying to satisfy the linter. I've put it on `ignore` now.